nemo_text_processing/inverse_text_normalization/hi/data/whitelist/whitelist_time.tsv

```diff
@@ -0,0 +1,2 @@
+१:३०	डेढ़
```

Comment:
why is this whitelist and not time?

Reply:
These entries, such as 'dedh' and 'dhai' for time, represent specific times that have no alternative expressions. They are fixed and unique terms. Therefore, as per my understanding and knowledge, I’ve included them in the whitelist folder, as they will be handled there once the whitelist is developed.